src/p11_child/p11_child_openssl.c

```diff
@@ -1847,16 +1848,19 @@ errno_t do_card(TALLOC_CTX *mem_ctx, struct p11_ctx *p11_ctx,
          * the requested ones for pre-auth. For authentication all attributes
          * must be given and match, for pre-auth only the given ones must
          * match. */
-        DEBUG(SSSDBG_TRACE_ALL, "%s %s %s %s %s %s.\n",
+        DEBUG(SSSDBG_TRACE_ALL, "%s %s %s %s %s %s %s.\n",
               module_name_in, module_file_name, token_name_in, token_name,
-              key_id_in, item->id);
+              key_id_in, label_in, item->id);
```

Comment:
`main()` doesn't ensure `label_in != NULL` so I think it's unsafe to provide it as `%s` without check.

Reply:
I think recent libc version can handle `%s` with `NULL` just fine, but I added a check to be on the safe side.